kie-wb-tests/kie-wb-tests-gui/src/test/java/org/kie/wb/selenium/ui/StandalonePerspectivesIntegrationTest.java

```diff
@@ -0,0 +1,121 @@
+            put("PreferencesCentralPerspective", By.className("preferences-editor"));
+            put("ProcessDefinitions", By.className("qe-list-bar-header-Process-Definitions"));
+            put("ProcessInstances", By.className("qe-list-bar-header-Process-Instances"));
+            put("Requests", By.className("qe-list-bar-header-Requests-List"));
+            put("SecurityManagementPerspective", By.className("qe-list-bar-header-Security-management"));
+            put("ServerManagementPerspective", By.className("fa-refresh"));
+            put("SocialHomePagePerspective", By.className("qe-list-bar-header-Recent-Assets"));
+            put("TaskAdmin", By.className("qe-list-bar-header-Task-Administration-List"));
+            put("Tasks", By.className("qe-list-bar-header-Tasks-List"));
+            put("UserHomePagePerspective", By.className("fa-home"));
+            put("WiresGridsDemoPerspective", By.className("qe-list-bar-header-Grids"));
+            put("WiresTreesPerspective", By.className("qe-list-bar-header-Palette"));
+        }
+    };
+
+    @Test
+    public void loadStandalonePerspectives() {
+        login.getLoginPage();
+        login.loginDefaultUser();
+
+        for (Map.Entry<String, By> entry : perspectiveToElement.entrySet()) {
+            String perspectiveId = entry.getKey();
+            By elementInPerspective = entry.getValue();
+
+            loadPerspectiveAndCheckLoadingTime(perspectiveId, true);
+            verifyPerspectiveIsLoaded(perspectiveId, elementInPerspective);
+            verifyPresenceOfHeader(true);
+
+            loadPerspectiveAndCheckLoadingTime(perspectiveId, false);
+            verifyPerspectiveIsLoaded(perspectiveId, elementInPerspective);
+            verifyPresenceOfHeader(false);
+        }
+    }
+
+    private void loadPerspectiveAndCheckLoadingTime(String perspectiveID, boolean headerIncluded) {
+        //long loadingStart = System.currentTimeMillis();
+
+        String standalonePerspectiveURL = getStandalonePerspectiveURL(perspectiveID, headerIncluded);
+        driver.get(standalonePerspectiveURL);
+
+        BusyPopup.waitForDisappearance();
+        ApplicationLoadingPopup.waitForDisappearance(PERSPECTIVE_LOADING_TIMEOUT);
+
+        //double perspectiveLoadingTime = (System.currentTimeMillis() - loadingStart) / (double) 1000;
+
+        //TODO: What is the acceptable loading time? Do we want to test this?
+        //assertThat(perspectiveLoadingTime)
+        //        .as("It should take less than "+PERSPECTIVE_LOADING_TIMEOUT+" seconds to load the perspective " + perspectiveID + ".")
+        //        .isLessThan(PERSPECTIVE_LOADING_TIMEOUT);
+    }
+
+    private void verifyPerspectiveIsLoaded(String perspectiveID, By elementInPerspective) {
+        try {
+            Waits.elementPresent(elementInPerspective, 10);
+        } catch (WebDriverException ex) {
+            Assertions.fail("The perspective with ID " + perspectiveID + " could not be loaded.");
```

Comment:
I forgot we should add `ex` as the second parameter of `Assertions.fail` not to loose the failure stacktrace. Please add it.